crates/kornia-io/src/error.rs

```diff
@@ -10,35 +10,35 @@ pub enum IoError {
     InvalidFileExtension(std::path::PathBuf),
 
     /// Error to open the file.
-    #[error("Failed to manipulate the file")]
+    #[error("Failed to manipulate the file. {0}")]
```

Comment:
all this errors we should derive from `transparent` so that we forward directly the original error messages unless we want to override some of them